src/ontology/cl-edit.owl

```diff
@@ -31277,6 +31279,28 @@ AnnotationAssertion(rdfs:label obo:CL_4033037 "mucus secreting cell of tracheobr
 EquivalentClasses(obo:CL_4033037 ObjectIntersectionOf(obo:CL_0000319 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0007196) ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_8410077)))
 SubClassOf(obo:CL_4033037 obo:CL_0002202)
 
+# Class: obo:CL_4033038 (lung resident memory CD4-positive, alpha-beta T cell)
```

Comment:
As noted in [this comment](https://github.com/obophenotype/cell-ontology/issues/1956#issuecomment-1523769252), this synonym should be added:
"lung resident memory CD4 T cell" (exact, PMID:[24064670](https://pubmed.ncbi.nlm.nih.gov/24064670))

Also consider:

‘lung-resident memory CD4-positive, alpha-beta T cell'

to reflect the hyphen that should be present in the compound adjective.